### PR TITLE
Update JSON Writes for names, forename and secondForename

### PR DESCRIPTION
**Pull request by Jxr227**

### CODE REVIEW

File: app/v1/models/request/NinoApplication.scala [hunk 1]
```diff
@@ -84,7 +84,7 @@ object NinoApplication {
       officeNumberPath.read[String].filter(commonError("office number"))(validateAgainstRegex(_, officeNumberRegex)) and
       contactNumberPath.readNullable[String].filter(commonError("contact number"))(_.fold(true)(validateAgainstRegex(_, contactNumberRegex))) and
       countryPath.read[Int] and
-      namePath.read[NameModel] and
+      namePath.read[NameModel].map(Seq(_)) and
```
Comment (TomDudley92) [inline app/v1/models/request/NinoApplication.scala]: Can you update namePath to be `names` please

Comment (Jxr227) [reply app/v1/models/request/NinoApplication.scala]: Done

File: app/v1/models/request/NinoApplication.scala [hunk 2]
```diff
@@ -62,7 +62,7 @@ object NinoApplication {
   private val officeNumberPath = __ \ "officeNumber"
   private val contactNumberPath = __ \ "contactNumber"
   private val countryPath = __ \ "country"
-  private val namePath = __ \ "name"
+  private val namesPath = __ \ "name"
```
Comment (TomDudley92) [inline app/v1/models/request/NinoApplication.scala]:
```suggestion
  private val namesPath = __ \ "names"
```
sorry if I wasn't clear in my first comment

Comment (TomDudley92) [reply app/v1/models/request/NinoApplication.scala]: Ignore this. This will be done under another story

**Pull request merged**